manifest.go

```diff
@@ -17,17 +17,17 @@ import (
 const ManifestName = "Gopkg.toml"
 
 type Manifest struct {
-	Dependencies gps.ProjectConstraints
-	Ovr          gps.ProjectConstraints
-	Ignored      []string
-	Required     []string
+	Constraints gps.ProjectConstraints
+	Ovr         gps.ProjectConstraints
+	Ignored     []string
+	Required    []string
 }
 
 type rawManifest struct {
-	Dependencies []rawProject `toml:"dependencies,omitempty"`
-	Overrides    []rawProject `toml:"overrides,omitempty"`
-	Ignored      []string     `toml:"ignored,omitempty"`
-	Required     []string     `toml:"required,omitempty"`
+	Constraints []rawProject `toml:"constraint,omitempty"`
+	Overrides   []rawProject `toml:"overrides,omitempty"`
```

Comment:
As long as we're in here changing things, let's also change `[[overrides]]` to the singular, `[[override]]` (but keep the property name plural, `rawManifest.Overrides`).